### DISCUSSION TITLE
How to perform a custom sorting on specific column?

### DISCUSSION THREAD

**Discussion by unnatRevo** [l-lin/angular-datatables#1662] [Q&A]
I have been using angular-datatable and it has been working fine so lately. Now I want to do sorting on a specific column on click of the column header. I googled a bit but was not able to find any solution. Can anyone please let me know how to achieve this functionality?

**Edit:**
If I need to develop a plugin to achieve this functionality, can anyone let how to develop plugin for angular-datatable?

**Comment by shanmukhateja** [comment]
Hello,

Column sorting works out of the box in datatables.net library. You don't need to develop plugins for it.

Are you using server-side processing? If so, you can send the column name (to be filtered) and the sort order (ascending or descending) to the server and let the backend perform the sort and return back the results.

Here's a reference for server-side processing: https://datatables.net/manual/server-side#Sent-parameters

If this didn't help, can you provide a reproducible StackBlitz repo to investigate this?
